test/handle_access_token_response_test.dart

```diff
@@ -47,6 +47,12 @@ void main() {
       expect(() => handleError(headers: {
         'content-type': 'text/plain'
       }), throwsFormatException);
+    }, skip: 'text/plain content-type is accepted now');
```

Comment:
Why is `text/plain` accepted?

Also, marking tests skipped is for temporarily getting the build green while working on a fix. If a test is no longer correct, it should be fixed or deleted.